src/Service/ViewHelperService.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace App\Service;
+
+use App\Entity\View;
+use App\Form\ViewAddStepOneType;
+use App\Form\ViewAddStepThreeType;
+use App\Form\ViewAddStepTwoType;
+
+class ViewHelperService
+{
+    /**
+     * Supply the multistep information for create form.
+     *
+     * @return array[]
+     */
+    public function getCreateFormMultiSteps()
```

Comment:
I think these two functions should just be private functions in the ViewController that use them. No need for a service for them.